po4a

```diff
@@ -1492,6 +1498,17 @@ if (not $po4a_opts{"no-translations"}) {
             $doc->{TT}{po_in} = $po;
             $doc->{TT}{po_in}->stats_clear();
 
+	    my $translationBackupfile;
+	    if ($po4a_opts{"no-rm-translations"} &&
+		-e $document{$master}{$lang}) {
+		(undef,$translationBackupfile)=
+		    File::Temp::tempfile("po4a-transBackupXXXX",
```

Comment:
Could you use a better name, something like $existingName.bak or something?

Reply:
That was actually what I did originally, but that is not production grade. If you mess with files in place you risk that someone is already using that name, better to avoid it. The tempfilename however is not something I'm proud of.

Reply:
But unfortunately, with your schema, you loose the connexion between temp file and master file it comes from. Could we use File::Temp::tempfile("${masterfile}.bak-XXXX" maybe?